eventhubs/data-plane/azure-eventhubs-eph/pom.xml

```diff
@@ -45,6 +45,12 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+    <dependency>
+    	<groupId>com.microsoft.azure</groupId>
```

Comment:
We try to add new dependencies in parent pom.xml files so that versions are consistent between client libraries. You would add it here:
https://github.com/Azure/azure-sdk-for-java/blob/master/parent/pom.xml

@JonathanGiles Is this the correct place to define new dependencies? I tried to find gson in it, but doesn't look like it is declared there.